docs/collecting-kubernetes-metrics.md

```diff
@@ -1,4 +1,4 @@
-# Collecting Kubernetes Metrics
+# Collecting Kubernetes metrics
```

Comment:
Titles should be capitalized https://www.google.com/search?q=title+capitalization

Reply:
what about subheaders? :P